bin/elm-petstore.sh

```diff
@@ -27,6 +27,6 @@ fi
 
 # if you've executed sbt assembly previously it will use that instead.
 export JAVA_OPTS="${JAVA_OPTS} -XX:MaxPermSize=256M -Xmx1024M -DloggerPath=conf/log4j.properties"
-ags="generate -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g elm -o samples/client/petstore/elm $@"
+ags="generate -i modules/openapi-generator/src/test/resources/2_0/petstore.yaml -g elm -t modules/openapi-generator/src/main/resources/elm -o samples/client/petstore/elm $@"
```

Comment:
do we need this?

Reply:
Yes, I think so as using -t allows us to regenerate the samples without doing another "mvn clean package" after updating the templates.

Reply:
Yes, I also checked that it's used in other scripts.

Reply:
It feels a bit weird that it sort of overloads the build, but it makes sense to add it if that's also the case for the other generators.